roles/principal_technical_consultant.md

```diff
@@ -0,0 +1,100 @@
+
+Client Principals will often pull Principal Technical Consultants into early conversations with potential customers to be the technical authority in such dialog.
+
+When it makes sense, our Principal Technical Consultants help define high level roadmaps that can then be turned into deliverable programmes of work by a Made Tech delivery team. They will remain available throughout the delivery in an advisory role to the team.
+
+## What experience are we looking for?
+
+We expect you to have experience managing a wide range of customer stakeholders in consultative engagements. You will be able to run meetings and workshops with stakeholders including C-level execs, IT Leadership and other business stakeholders.
+
+We expect you to have experience in delivering digital and technology outcomes, both in build and buy scenarios, and able to coordinate customer stakeholders, delivery teams and third party suppliers where appropriate.
+
+We expect you to have experience in delivering digital transformation. You need to be able to assess digital capability and drive organisational change.
+
+We expect you to have experience with enterprise architecture, both understanding existing landscapes and designing new ones. You need to be able to run workshops both with customers and our own software delivery teams. We expect you to have significant experience in designing and developing web applications in modern cloud-based environments.
+
+Business savvy is a must. You will need experience in translating business goals and issues into technical solutions and have success stories to share. You will need to demonstrate experience working with senior IT and business stakeholders to create technical strategies.
+
+## What is it like to work at Made Tech?
+
+We are a sanctuary for those wanting to hone their skills alongside like-minded learners. After joining our team it is common for new starters to comment on how much they’re learning and how much they enjoy the fact they are surrounded by people they can learn from. This includes our most senior hires.
+
+The biggest thing you’ll take away from our culture after spending the day with us would be that we practice continuous improvement at every level. Everyone has peer-based one-to-ones every 2 weeks, along with monthly one-to-ones with a Line Manager. Teams have fortnightly retrospectives. We also hold a company wide retrospective fortnightly. We discuss our problems out in the open, and rather than punish failure we band together to find solutions.
+
+Other notable things:
+
+- Every Friday afternoon is dedicated to learning new skills
+- Everyone is encouraged to write blog posts regularly
+- Our handbook is open sourced
+- We are vegan and non-drinker friendly as well as meat-eater and drinker friendly
+- Retreats and trips every year
+
+## Salary
+
+This role has a salary of £85-95k depending on experience.
+
+
+## Expectations
```

Comment:
Should there be clearer expectations around 'buy' ecosystem? I would expect somebody in this role to be able to provide impartial advice around options. e.g. Salesforce is an option, as it provides X, Y and Z results.